lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -377,6 +379,27 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {LH.Audit.Details.SourceLocationValue} item
+   * @return {Element}
+   * @protected
+   */
+  renderSourceLocation(item) {
+    let element;
+    if (item.urlIsNetworkResource) {
+      element = this.renderTextURL(item.url);
+      this._dom.find('a', element).textContent += `:${item.line + 1}:${item.column}`;
```

Comment:
Is there a reason this is `+1` to item.line (if there is comment plz)?  Seems odd that it would be adjusted here, and not just +1'd from the beginning? It def makes the tests look weird where line#10 is confirmed to = 11.

Reply:
Chrome and DevTools expected a zero indexed number for lines and columns. However, an IDE (or any view, like a stack trace) would never show "line 0".